static/app/views/settings/project/projectOwnership/codeowners.tsx

```diff
@@ -52,7 +52,7 @@ class CodeOwnersPanel extends Component<Props> {
             dateUpdated={dateUpdated}
             provider={provider}
             repoName={repoName}
-            readOnly
```

Comment:
Is there a reason we aren't showing the  "read only" badge anymore? Can we not have both?

Reply:
@MeredithAnya the current design doesn't have it. I agree we should have it, but it looks a little crowded in the Code Owners panel. We can have the design team iterate on this.

Reply:
okay cool just wanted the context on that. seems fine for EA and we'll see what feedback we get